student.rb

```diff
@@ -0,0 +1,19 @@
+require_relative 'person'
+
+class Student < Person
+  attr_accessor :classroom
+
+  def initialize(classroom: nil, name: 'Unknown', age: 0, parent_permission: true)
+    super(name: name, age: age, parent_permission: parent_permission)
+    @classroom = classroom
+  end
+
+  def play_hooky
+    '¯\\(ツ)/¯'
+  end
+
+  def add_a_classroom(classroom)
+    @classroom = classroom
+    classroom.adding_student(self)
```

Comment:
- [ ] Here you are adding a new student in the classroom I suggest that you make it:
`classroom.adding_student.push(self) unless classroom.adding_student.include?(self)`